src/app/technical-record/dimensions/__snapshots__/dimensions.component.spec.ts.snap

```diff
@@ -99,32 +99,7 @@ exports[`DimensionsComponent should create view only with populated data 1`] = `
       
       
       
-      <tr
-        class="govuk-table__row"
-      >
-        <th
-          class="govuk-table__cell"
-        />
-        <td
-          class="govuk-table__cell govuk-!-font-weight-bold"
-        >
-          Minimum (mm)
-        </td>
-        <td
-          class="govuk-table__cell govuk-!-font-weight-bold"
-        >
-          Maximum (mm)
-        </td>
-        <td
-          class="govuk-table__cell"
-        />
-        <td
-          class="govuk-table__cell"
-        />
-        <td
-          class="govuk-table__cell"
-        />
-      </tr>
+      
```

Comment:
Umer let update the existing `should create view only with populated data` with these 2 tests
- should create view only with HGV or TRL populated data
- should create view only with other vehicle types

Reply:
done